src/components/utils/onfidoApi.js

```diff
@@ -3,44 +3,41 @@ import Tracker from '../../Tracker'
 import forEach from 'object-loops/for-each'
 import { humanizeField } from '../utils'
 
-const formatError = ({response, status}) => ({status, response: JSON.parse(response)})
+const formatError = ({response, status}, onError) =>
+  onError({status, response: JSON.parse(response)})
 
-export const postToOnfido = ({blob, documentType, side}, captureType, token, advanced_validation, onSuccess, onError) => {
-  if (captureType === 'face') {
-    return sendFile(
-      {blob},
-      'live_photos',
-      token,
-      onSuccess,
-      request => onError(formatError(request))
-    )
-  }
-  sendFile(
-    {blob, type: documentType, side, advanced_validation},
-    'documents',
-    token,
-    onSuccess,
-    request => onError(formatError(request))
-  )
+export const uploadDocument = (data, token, onSuccess, onError) => {
+  const endpoint = `${process.env.ONFIDO_API_URL}/v2/documents`
+  sendFile(endpoint, data, token, onSuccess, onError)
+}
+
+export const uploadLivePhoto = (data, token, onSuccess, onError) => {
+  const endpoint = `${process.env.ONFIDO_API_URL}/v2/live_photos`
+  sendFile(endpoint, data, token, onSuccess, onError)
 }
 
+
 const objectToFormData = (object) => {
   const formData = new FormData()
-  forEach(object, (value, key) => formData.append(key, value))
+  forEach(object, (value, key) => {
+    const isObject = (value instanceof Object) && !(value instanceof File)
```

Comment:
`typeof` a file object returns Object rather than File so I think we have to use `instanceof` in this case